.github/workflows/handle-release-branches.yml

```diff
@@ -0,0 +1,126 @@
+      - id: next-version
+        uses: notiz-dev/github-action-json-property@release
+        with:
+          path: ${{ github.workspace }}/next/docs/versions/next.json
+          prop_path: version
+          
+      - run: |
+          NEXT_RELEASE_BRANCH=($(echo ${{ steps.next-version.outputs.prop }} | sed -E 's/([0-9]+)\.([0-9]+).*/release-\1-\2/'))
+          echo "next-release-branch=$NEXT_RELEASE_BRANCH" >> $GITHUB_ENV
+    outputs:
+      branch: ${{ env.next-release-branch }}
+  
+  create-next-release-branch:
+    needs: [branch-checks, get-next-release-branch]
+    if: ${{ needs.branch-checks.outputs.is-next-branch == 'true' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          git branch temp
+          git push -f origin temp:${{ needs.get-next-release-branch.outputs.branch }}
+    outputs:
+      branch: ${{ needs.get-next-release-branch.outputs.branch }}
+  
+  latest-or-next-release-branch-check:
+    if: ${{ always() }}
+    needs: [branch-checks, get-latest-release-branch, get-next-release-branch]
+    runs-on: ubuntu-latest
+    steps:
+      - run: |
+          IS_LATEST_RELEASE_BRANCH=${{ needs.branch-checks.outputs.branch == needs.get-latest-release-branch.outputs.branch }}
+          IS_NEXT_RELEASE_BRANCH=${{ needs.branch-checks.outputs.branch == needs.get-next-release-branch.outputs.branch }}
+          echo "is-latest-release-branch=$IS_LATEST_RELEASE_BRANCH" >> $GITHUB_ENV
+          echo "is-next-release-branch=$IS_NEXT_RELEASE_BRANCH" >> $GITHUB_ENV
+          
+      - run: |
+          IS_LATEST_OR_NEXT_RELEASE_BRANCH=${{ env.is-latest-release-branch == 'true' || env.is-next-release-branch == 'true' }}
+          echo "is-latest-or-next-release-branch=$IS_LATEST_OR_NEXT_RELEASE_BRANCH" >> $GITHUB_ENV      
+      
+      - if: ${{ env.is-latest-release-branch == 'true' }}
+        run: echo "relevant-base-branch=main" >> $GITHUB_ENV
+        
+      - if: ${{ env.is-next-release-branch == 'true' }}
+        run: echo "relevant-base-branch=next" >> $GITHUB_ENV
+      
+      - if: ${{ env.is-latest-or-next-release-branch == 'true' }}
+        run: |
+          echo "WARNING: Do not push directly to the `${{ needs.branch-checks.outputs.branch }}` branch. This branch is created and force-pushed over after pushing to the `${{ env.relevant-base-branch }}` branch and the changes you just pushed will be lost."
+          exit 1
+    outputs:
+      check: ${{ env.is-latest-or-next-release-branch }}
+  
+  request-create-frontpage-branch:
+    if: ${{ always() }}
+    needs: [branch-checks, latest-or-next-release-branch-check, create-latest-release-branch, create-next-release-branch]
+    runs-on: ubuntu-latest
+    steps:
+      - if: ${{ needs.branch-checks.outputs.is-actionable-branch == 'true' && needs.latest-or-next-release-branch-check.outputs.check == 'false' }}
+        run: |
+          curl -X POST https://api.github.com/repos/storybookjs/frontpage/dispatches \
```

Comment:
This technique is from https://blog.marcnuri.com/triggering-github-actions-across-different-repositories.

The `frontpage` workflow would look like:

```yml
name: Respond to request-create-frontpage-branch

on: [repository_dispatch]

jobs:
  build:
    runs-on: ubuntu-latest
    if: github.event.action == 'request-create-frontpage-branch'
    steps:
      - uses: actions/checkout@v2
      - run: |
          echo "Creating `${{ github.event.client_payload.branch }}` branch..."
          git branch temp
          git push -f origin temp:${{ github.event.client_payload.branch }}
          curl -X POST https://api.github.com/repos/kylegach/actions-test-a/dispatches \
          -H 'Accept: application/vnd.github.v3+json' \
          -u ${{ secrets.ACCESS_TOKEN }} \
          --data '{"event_type": "created-frontpage-branch", "client_payload": { "branch": "${{ needs.branch-checks.outputs.branch }}" }}'
```

Pushing the `release-x-x` branch would then kick off a Netlify build for that branch, which would pull in the appropriate version's content from this repo's `release-x-x` branch.

The `curl` above would allow for _this_ repo to receive a confirmation message, if desired. Otherwise, it can be removed.